src/components/AnonymousChat/child/ChatItemTargetText.tsx

```diff
@@ -71,27 +79,42 @@ const ChatItemTargetText = ({
   time = '4h',
   isContinuous = false,
   message = '',
+  attachments = [],
   avatar
 }: ChatItemMyTextProps) => {
   const renderAvatar = React.useCallback(() => {
     if (isContinuous) return <View style={styles.avatar} />;
     return <View style={styles.mr8}>{avatar}</View>;
   }, []);
 
+  const handleTextContainerStyle = () => {
+    const isAttachment = attachments.length > 0;
+    const paddingStyle = {
+      paddingLeft: isAttachment ? BUBBLE_LEFT_PADDING_ATTACHMENT : BUBBLE_LEFT_PADDING,
+      paddingRight: isAttachment ? BUBBLE_RIGHT_PADDING_ATTACHMENT : BUBBLE_RIGHT_PADDING
+    };
+    return [styles.textContainer, paddingStyle];
+  };
+
   return (
     <View style={styles.chatContainer}>
       {renderAvatar()}
-      <View style={styles.textContainer}>
+      <View style={handleTextContainerStyle()}>
         {!isContinuous && (
-          <View testID="chat-item-user-info" style={styles.chatTitleContainer}>
+          <View
+            testID="chat-item-user-info"
+            style={[styles.chatTitleContainer, attachments.length > 0 ? {marginBottom: 4} : {}]}>
             <Text style={styles.userText}>{username}</Text>
             <View style={styles.dot} />
             <Text style={styles.timeText}>{time}</Text>
           </View>
         )}
-        <Text testID="chat-item-message" style={styles.text}>
-          {message}
-        </Text>
+        {attachments.length > 0 && <ChatItemAttachment attachments={attachments} />}
+        {attachments.length <= 0 && (
+          <Text testID="chat-item-message" style={styles.text}>
+            {message}
+          </Text>
+        )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces handling for attachments in chat messages, which was not present in the old code. It's a significant improvement in terms of functionality. The use of conditional rendering based on whether attachments exist or not is a good practice. However, ensure that the `ChatItemAttachment` component handles potential errors gracefully, such as when an attachment fails to load.

Also, consider extracting the logic inside `handleTextContainerStyle` into a useMemo hook to avoid unnecessary computations on each render.
```diff
+  const handleTextContainerStyle = React.useMemo(() => {
+    const isAttachment = attachments.length > 0;
+    const paddingStyle = {
+      paddingLeft: isAttachment ? BUBBLE_LEFT_PADDING_ATTACHMENT : BUBBLE_LEFT_PADDING,
+      paddingRight: isAttachment ? BUBBLE_RIGHT_PADDING_ATTACHMENT : BUBBLE_RIGHT_PADDING
+    };
+    return [styles.textContainer, paddingStyle];
+  }, [attachments]);
```
This way, the style will only be recalculated when the `attachments` prop changes.


<!-- This is an auto-generated comment by OSS CodeRabbit -->